### PR TITLE
Unit Tests for Home.vue

### PR DESCRIPTION
**Pull request by HeatherJoYoung**
Adds unit test for Home.vue

To run the tests, 
1. open a terminal/command prompt
2. cd into the concierge-web directory
3. `npm run test`

When you do that, all tests should pass

### CODE REVIEW

File: src/components/Home.vue [hunk 1]
```diff
@@ -66,9 +65,6 @@
       this.reservations[1].date = this.today.add(1, 'day')
     },
     computed: {
-      hasUserData() {
```
Comment (HeatherJoYoung) [inline src/components/Home.vue:69]: Removing this user data stuff, which is left over from when I first set up the app. It's not needed in the Home page now.

**Review by fritterj-fhsu — APPROVED** [review]
It seems like in a production environment we might want more tests, like testing edge cases?  I assume you would know better than me.  But this works and it gives me a model to follow in creating more of these.
